snippets/base/models.py

```diff
@@ -1933,16 +1941,6 @@ def analytics_export(self):
         }
         return export
 
-    def save(self, *args, **kwargs):
```

Comment:
I assume this means the `distributions` field will be going away in a subsequent migration? If there used to be multiple distributions per job, why is it moving to one distribution per job?

Reply:
Yes, `distributions` is going away as I note in the [notes of the issue](https://github.com/mozmeao/snippets-service/issues/1143). We move the ability of including jobs to multiple distributions up to the DistributionBundle level. A Bundle can have multiple Distributions and a Job one Distribution.